[2025-07-21T16:53:05.917+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: complex_failure_dag.long_running_task manual__2025-07-21T16:52:43.980676+00:00 [queued]>
[2025-07-21T16:53:05.923+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: complex_failure_dag.long_running_task manual__2025-07-21T16:52:43.980676+00:00 [queued]>
[2025-07-21T16:53:05.923+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 4
[2025-07-21T16:53:05.935+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): long_running_task> on 2025-07-21 16:52:43.980676+00:00
[2025-07-21T16:53:05.939+0000] {standard_task_runner.py:57} INFO - Started process 196 to run task
[2025-07-21T16:53:05.942+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'complex_failure_dag', 'long_running_task', 'manual__2025-07-21T16:52:43.980676+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/complex_failure_dag.py', '--cfg-path', '/tmp/tmp6plpyofc']
[2025-07-21T16:53:05.943+0000] {standard_task_runner.py:85} INFO - Job 20: Subtask long_running_task
[2025-07-21T16:53:05.977+0000] {task_command.py:416} INFO - Running <TaskInstance: complex_failure_dag.long_running_task manual__2025-07-21T16:52:43.980676+00:00 [running]> on host 2cf9a9b2cf2c
[2025-07-21T16:53:06.033+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='metadata_test' AIRFLOW_CTX_DAG_ID='complex_failure_dag' AIRFLOW_CTX_TASK_ID='long_running_task' AIRFLOW_CTX_EXECUTION_DATE='2025-07-21T16:52:43.980676+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-21T16:52:43.980676+00:00'
[2025-07-21T16:53:06.034+0000] {task_logging_plugin.py:29} INFO - [Task Metadata] (running): {'event': 'TASK_RUNNING', 'timestamp': '2025-07-21T16:53:06.034419', 'dag_id': 'complex_failure_dag', 'dag_run_id': 'manual__2025-07-21T16:52:43.980676+00:00', 'task_id': 'long_running_task', 'execution_date': '2025-07-21 16:52:43.980676+00:00', 'try_number': 1, 'start_date': '2025-07-21 16:53:05.917626+00:00', 'end_date': 'None', 'state': 'running', 'queue': 'default', 'operator': 'PythonOperator', 'hostname': '2cf9a9b2cf2c'}
[2025-07-21T16:53:06.034+0000] {task_logging_plugin.py:33} INFO - [Task Metadata] (running): {'event': 'TASK_RUNNING', 'timestamp': '2025-07-21T16:53:06.034419', 'dag_id': 'complex_failure_dag', 'dag_run_id': 'manual__2025-07-21T16:52:43.980676+00:00', 'task_id': 'long_running_task', 'execution_date': '2025-07-21 16:52:43.980676+00:00', 'try_number': 1, 'start_date': '2025-07-21 16:53:05.917626+00:00', 'end_date': 'None', 'state': 'running', 'queue': 'default', 'operator': 'PythonOperator', 'hostname': '2cf9a9b2cf2c'}
[2025-07-21T16:53:06.034+0000] {logging_mixin.py:154} INFO - Starting long running task...
[2025-07-21T16:54:21.110+0000] {logging_mixin.py:154} INFO - Long running task completed
[2025-07-21T16:54:21.111+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-07-21T16:54:21.130+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=complex_failure_dag, task_id=long_running_task, execution_date=20250721T165243, start_date=20250721T165305, end_date=20250721T165421
[2025-07-21T16:54:21.144+0000] {task_logging_plugin.py:29} INFO - [Task Metadata] (success): {'event': 'TASK_SUCCESS', 'timestamp': '2025-07-21T16:54:21.144356', 'dag_id': 'complex_failure_dag', 'dag_run_id': 'manual__2025-07-21T16:52:43.980676+00:00', 'task_id': 'long_running_task', 'execution_date': '2025-07-21 16:52:43.980676+00:00', 'try_number': 2, 'start_date': '2025-07-21 16:53:05.917626+00:00', 'end_date': '2025-07-21 16:54:21.130855+00:00', 'state': <TaskInstanceState.SUCCESS: 'success'>, 'queue': 'default', 'operator': 'PythonOperator', 'hostname': '2cf9a9b2cf2c'}
[2025-07-21T16:54:21.144+0000] {task_logging_plugin.py:33} INFO - [Task Metadata] (success): {'event': 'TASK_SUCCESS', 'timestamp': '2025-07-21T16:54:21.144356', 'dag_id': 'complex_failure_dag', 'dag_run_id': 'manual__2025-07-21T16:52:43.980676+00:00', 'task_id': 'long_running_task', 'execution_date': '2025-07-21 16:52:43.980676+00:00', 'try_number': 2, 'start_date': '2025-07-21 16:53:05.917626+00:00', 'end_date': '2025-07-21 16:54:21.130855+00:00', 'state': <TaskInstanceState.SUCCESS: 'success'>, 'queue': 'default', 'operator': 'PythonOperator', 'hostname': '2cf9a9b2cf2c'}
[2025-07-21T16:54:21.165+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-07-21T16:54:21.189+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
